composer.json

```diff
@@ -79,7 +79,6 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "^3.155",
-        "doctrine/dbal": "^2.6|^3.0",
```

Comment:
You still need to re-add this in require-dev.

Reply:
@driesvints check now